rules/rules.js

```diff
@@ -108,6 +108,55 @@ const removeRule = function (uid) {
     }
 };
 
+/**
+ * Runs the rule with the given UID. Throws errors when the rule doesn't exist
+ * or is unable to run (e.g. it's disabled);
```

Comment:
I think the the semicolon is a typo and should be a point.

Reply:
Note that 109 is not among my changes. That was there to begin with.

I do notice it's not consistent in how it's used in this file so I can change them all to be consistent.

I don't think `.` is right though. It really is the end of a statement, the assignment of the function to `removeRule`.

Reply:
I think we are not talking about the same ;-)

> I don't think . is right though. It really is the end of a statement, the assignment of the function to removeRule.

That’s correct.

I mean the semicolon at the end of the comment in line 113.